### PR TITLE
Add smallest OpenAPI schema 3.0 as .yaml and .json

### PR DESCRIPTION
**Pull request by maxpatiiuk**
Add smallest valid OpenAPI schema 3.0 files as `.yaml` and `.json`.

### CODE REVIEW

**Comment by maxxxxxdlp** [comment]
Files are considered valid according to [editor.swagger.io](https://editor.swagger.io/).

**Comment by dscorbett** [comment]
Some of the spaces in openapi.yaml can be deleted. The newlines at the ends of both files can probably be deleted too.

**Review by mathiasbynens — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+{openapi: 3.0.0, info: {title: '', version: ''}, paths: {/: {get: {responses: {'204': {description: ''}}}}}}
```
Comment (mathiasbynens) [inline openapi.yaml:1]: The spaces?

Comment (maxxxxxdlp) [reply openapi.yaml:1]: I think those spaces are required as the following fails to parse in the Swagger Editor:
```yaml
{openapi:3.0.0,info:{title:'',version:''},paths:{/:{get:{responses:{'204':{description:''}}}}}}
```
It also fails under this editor: https://mermade.github.io/openapi-gui/

Comment (dscorbett) [reply openapi.yaml:1]: Some of them are required but most aren’t.

Comment (maxxxxxdlp) [reply openapi.yaml]: Oh, I see. Check out the latest commit. I think I removed all the spaces that aren't required.

**Pull request merged**